lib/signs/utilities/headways.ex

```diff
@@ -58,6 +82,17 @@ defmodule Signs.Utilities.Headways do
       }}}
   end
 
+  @spec get_paging_headway_message(
+          PaEss.destination() | nil,
+          Engine.Config.Headway.t()
+        ) :: Content.Message.Headways.Paging.t()
+  defp get_paging_headway_message(destination, headways) do
+    %Content.Message.Headways.Paging{
+      destination: destination,
+      range: {headways.range_low, headways.range_high}
```

Comment:
When I tested locally, this line was giving errors because `headways` was nil in some cases. Tracing back, this value comes from `Engine.Config.Headways.get_headway` which can sometimes return nil, so we'll need to guard against that, either here or in `get_paging_message` above.

Reply:
Ah yeah that's right, this will throw errors unless there are actual headway values in the config which we should handle, just in case. And also for local development